app/src/app/components/contents/documents/ProjectDocumentation.tsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+export default function ProjectDocumentation() {
+    return (
+        <div>Project documentation page</div>
+    );
+}
```

Comment:
Could you just make sure that all files have a new line at the end,an we an an `eslint` rule for this?
In `.eslintrc.json`, under "rules" add:
`"eol-last": 2`